chain/chain/src/chain.rs

```diff
@@ -4197,9 +4200,16 @@ impl Chain {
                 "existing_chunk",
                 shard_id)
             .entered();
+            let storage_config = RuntimeStorageConfig {
+                state_root: *new_extra.state_root(),
+                use_flat_storage: true,
```

Comment:
I'm curious - I know this was the behavior before too, but how come we can just hardcode true here - don't we need to keep compatibility with the behavior before flat storage was launched?

Reply:
Ohhh, gotcha. Thanks!